wolox-books/src/views/SignUp.vue

```diff
@@ -11,18 +11,25 @@
     input.input(type='text' v-model='lastName')
     label.label
       | Email
-    input.input(type='text' v-model='email')
+    input.input(type='text' v-model='$v.email.$model')
+    .error(v-if="$v.email.$error")
+      | Email is required
     label.label
       | Password
-    input.input(type='password' v-model='password')
-    a.input.signup-button(href='javascript:void(0)' v-on:click='submitForm')
-      | Sign Up
+    input.input(type='password' v-model='$v.password.$model')
+    .error(v-if="$v.password.$error")
+      | Password is required
+    input.input.signup-button(v-on:click='submitForm' type='submit' value='Sign Up')
     .login-container
```

Comment:
This one mustn't be a div.

Reply:
you told me to use a div to put a border between buttons 👀

Reply:
Oh, is it being used for that? Then, ignore this comment.